.gitignore

```diff
@@ -0,0 +1,2 @@
+.style/style.css
```

Comment:
it dind't work. You still have same files in this PR. You need to fix these lines and remove `style.css`, `style.css.map` manually